### ISSUE TITLE
Program not terminating

### ISSUE THREAD

**Issue by cryarchy** [jaggedsoft/node-binance-api#130]
To replicate

```javascript
const binance = require('node-binance-api');
const config = require('./config');

binance.options({
  APIKEY: config.key,
  APISECRET: config.secret,
  test: true // Tried with this option missing. Same results
});
```
**Description:** The program does not terminate after setting of the binance opt.
**Node Version:** v8.2.1
The issue occurs with later versions of *node-binance-api*. I am unsure if the behavior is intended or it is a bug.

**Comment by jaggedsoft** [comment]
Sorry, my fault. This bug was already fixed, but I forgot to publish it

Published, available in v0.4.18
Fix endless running program https://github.com/jaggedsoft/node-binance-api/commit/a84999d04f80dd2549244b38d58d7a0c4ad34e00
Thank you @dreyescat for the solution!!!

**Comment by cryarchy** [comment]
Thanks. Works fine now.
